doc/Getting_Started_Writing_Clients.adoc

```diff
@@ -73,6 +73,10 @@ The servers that Yubico provides are:
 
 These servers are hosted in different places and by different organizations.
 
+Clients should send authentication requests to all of them in parallel, and
+utilize the first response recieved for added reliability should one or more
```

Comment:
A technical detail here is that the first response for a valid request might be a REPLAYED_REQUEST due to network latency etc. I believe awaiting the first non-REPLAYED_REQUEST response is the correct behavior (or until all requests have returned, and then failing as the request truly was replayed).